features/profile_groupe.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Profile group example
```

Comment:
I plan to merge it with the [profile.feature](https://github.com/denislaliberte/rspec-core/blob/1878_wip_7/features/configuration/profile.feature) when it's done.